inference_perf/loadgen/load_generator.py

```diff
@@ -182,23 +182,23 @@ async def run(self, client: ModelServerClient) -> None:
 
         for stage_id, stage in enumerate(self.stages):
             timer = self.get_timer(stage.rate)
-            start_time = time.perf_counter()
+            start_time = time.time()
             end_time = start_time + stage.duration
             logger.info("Stage %d - run started", stage_id)
             async with TaskGroup() as tg:
                 for _, (data, time_index) in enumerate(
                     zip(self.datagen.get_data(), timer.start_timer(start_time), strict=True)
                 ):
-                    now = time.perf_counter()
+                    now = time.time()
                     if time_index < end_time and now < end_time:
                         if time_index > now:
-                            await sleep(time_index - time.perf_counter())
+                            await sleep(time_index - time.time())
                         tg.create_task(client.process_request(data, stage_id, time_index))
                         continue
                     else:
                         break
             self.stage_runtime_info[stage_id] = StageRuntimeInfo(
-                stage_id=stage_id, rate=stage.rate, start_time=start_time, end_time=time.perf_counter()
+                stage_id=stage_id, rate=stage.rate, start_time=start_time, end_time=time.time()
```

Comment:
if stage_runtime_info only needs epoch time, we can just add a stage_start_time_epoch variable to be used when recording the stage_runtime_info instead of changing all time behavior.

Reply:
originally considered that too, problem is `start_time=time.perf_counter()` and `end_time=time.perf_counter()` on their own are meaningless, would make sense if we need greater than microsecond precision